src/compiler/diagnosticMessages.json

```diff
@@ -5824,6 +5824,11 @@
         "category": "Message",
         "code": 6929
     },
+    "`false` unless target is ESNext or above ES2022": {
+        "category": "Message",
+        "code": 6930
+    },
+

```

Comment:
I don't know how much you care about such things, but this seems to be the only occurrence of an empty newline in this file. One already exists, this PR adds a second.